src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -443,5 +470,70 @@ private static byte GetSeverityNumber(LogLevel logLevel)
                 return 1;
         }
     }
+
+    private static string Sanitize(string categoryName)
+    {
+        Span<char> tempArrSpan = stackalloc char[200];
+        int readIdx = 0;
+        int writeIdx = 0;
+
+        if (categoryName.Length > 0)
+        {
+            var firstChar = categoryName[0]; // special treatment for the first character.
+            if (firstChar >= 'A' && firstChar <= 'Z')
+            {
+                tempArrSpan[writeIdx] = firstChar;
+                ++writeIdx;
+            }
+            else if (firstChar >= 'a' && firstChar <= 'z')
+            {
+                // If the first character in the resulting string is lower-case ALPHA,
+                // it will be converted to the corresponding upper-case.
+                tempArrSpan[writeIdx] = (char)(firstChar - 32);
+                ++writeIdx;
+            }
+
+            // Not a valid name - Part B name should follow PascalCase naming convention.
+            else
+            {
+                return null;
+            }
+
+            ++readIdx;
+        }
+
+        while (readIdx < categoryName.Length)
+        {
+            // Recommended by Part B that table name should not be longer than 50 characters.
+            if (writeIdx >= 49)
```

Comment:
Suggestion: Move this to a constant. Something like...

```csharp
const stackAllocatedCharCount = 50;
...
Span<char> tempArrSpan = stackalloc char[stackAllocatedCharCount];
...
if (writeIdx >= stackAllocatedCharCount- 1)
```

Reply:
Thank you for your suggestion.
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/8ab761d635112f7640fcf589e5f873cedf4c2724/src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs#:~:text=const%20int%20maxNameCharCount,%7D